samples/kvsWebRTCClientViewer.c

```diff
@@ -2,6 +2,31 @@
 
 extern PSampleConfiguration gSampleConfiguration;
 
+// onMessage callback for a message received by the viewer on a data channel
+VOID dataChannelOnMessageCallback(UINT64 customData, PRtcDataChannel pDataChannel, BOOL isBinary, PBYTE pMessage, UINT32 pMessageLen)
+{
+    UNUSED_PARAM(customData);
+    STATUS retStatus = STATUS_SUCCESS;
```

Comment:
`retStatus` is unused so it can be removed.

Reply:
Looks like `pDataChannel` is unused as well.  If this is required to match the signature (I didn't check) then you can just mark it as `UNUSED_PARAM()`

Reply:
Marked `pDataChannel` as `UNUSED_PARAM()`. Removed `retStatus`.